### PR TITLE
Release SQL library on push to main

### PR DESCRIPTION
**Pull request by begelundmuller**

### CODE REVIEW

File: .github/workflows/sql-release.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          cd sql
+          mvn package -Pnative-lib
+      - name: Install zip on Windows
+        if: matrix.os == 'windows'
+        shell: bash
+        run: choco install zip -y --force
+      - name: Create archive
+        shell: bash
+        run: |
+          rm sql/target/*.txt
```
Comment (pjain1) [inline .github/workflows/sql-release.yml:56]: do we have any txt files or just being cautious ?

File: sql/pom.xml [hunk 2]
```diff
@@ -290,7 +290,7 @@
               <goal>exec</goal>
             </goals>
             <configuration>
-              <executable>/bin/bash</executable>
+              <executable>bash</executable>
```
Comment (pjain1) [inline sql/pom.xml:293]: why this change ? just need to understand

File: .github/workflows/sql-release.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            runner: ubuntu-latest
+          - os: windows
+            arch: amd64
+            runner: windows-latest
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          version: "22.1.0"
```
Comment (pjain1) [inline .github/workflows/sql-release.yml:27]: can we add note sql pom.xml that these needs to be updated if version are changed in there.

File: .github/workflows/sql-release.yml (hunk 1 above)
Comment (begelundmuller) [reply .github/workflows/sql-release.yml:56]: Yeah, GraalVM outputs a manifest `.txt` that we don't want to release

File: sql/pom.xml (hunk 2 above)
Comment (begelundmuller) [reply sql/pom.xml:293]: It [failed](https://github.com/rilldata/rill-developer/runs/8068489164?check_suite_focus=true) on Windows because the path to `bash` is different there

File: .github/workflows/sql-release.yml (hunk 3 above)
Comment (begelundmuller) [reply .github/workflows/sql-release.yml:27]: Done

File: .github/workflows/sql-release.yml (hunk 1 above)
Comment (begelundmuller) [reply .github/workflows/sql-release.yml]: I pushed an update that removes this, and more explicitly packs the zip archive

File: sql/pom.xml (hunk 2 above)
Comment (pjain1) [reply sql/pom.xml:294]: ok got it

**Review by pjain1 — APPROVED** [review]
👍 after travis
